### ISSUE TITLE
cannot generate unique device name

### ISSUE THREAD

**Issue by beyondkmp** [RobinLinus/snapdrop#203]
https://github.com/RobinLinus/snapdrop/blob/master/server/index.js#L203 should be 
```
displayName: uniqueNamesGenerator({ length: 2, separator: ' ', dictionaries: [adjectives, animals], style: 'capital' })
```

see https://github.com/RobinLinus/snapdrop/pull/91

**Comment by RobinLinus** [comment]
Thanks a lot. The issue should be fixed with 7469086cda5f762d4703dea18bcd5e4111b0c73b
Does that work for you?

**Comment by beyondkmp** [comment]
yes, thanks.
